src/Features/CSharp/Portable/Completion/CompletionProviders/ExplicitInterfaceMemberCompletionProvider.cs

```diff
@@ -106,20 +105,24 @@ public override async Task ProvideCompletionsAsync(CompletionContext context)
 
                 foreach (var member in members)
                 {
-                    if (member.IsAccessor() || member.Kind == SymbolKind.NamedType || !(member.IsAbstract || member.IsVirtual) ||
+                    if (member.IsAccessor() ||
+                        member.Kind == SymbolKind.NamedType ||
+                        !(member.IsAbstract || member.IsVirtual) ||
                         !semanticModel.IsAccessible(node.SpanStart, member))
                     {
                         continue;
                     }
 
-                    var displayText = member.ToMinimalDisplayString(
-                        semanticModel, namePosition, s_signatureDisplayFormat);
-                    var insertionText = displayText;
+                    var memberString = member.ToMinimalDisplayString(semanticModel, namePosition, s_signatureDisplayFormat);
+
+                    // Split the member string into two parts (generally the name, and the signature portion). We want
+                    // the split so that other features (like spell-checking), only look at the name portion.
+                    var (displayText, displayTextSuffix) = SplitMemberName(memberString);
```

Comment:
Does this change behavior in any way if there's multiple overloads exist and now the display text doesn't distinguish between them? It looks like we dismiss the list today if you type () and I can't get matching to match in the parameter names so this _might_ be fine, but just expressing concern.

Reply:
i checked, and tehre doesn't seem to be any issues here.